### ISSUE TITLE
[Question] How to run DFTD3 calculation after 4f08ae58d842cfafc8a6bf1144d19d4a478958b9 for PySCF built from source

### ISSUE THREAD

**Issue by zc62** [pyscf/pyscf#2397]
As of commit b27e4f9fb5fc3e33a93f4798194415d899b33511, I was able to run DFTD3 calculations after I install dftd3 via
```
conda install simple-dftd3 dftd3-python -c conda-forge
```
but after commit 4f08ae58d842cfafc8a6bf1144d19d4a478958b9, the dispersion part got separated as a plugin and requires `pip install pyscf-dispersion` to have that plugin, otherwise the code must terminates at https://github.com/pyscf/pyscf/blob/65ba28eec2623e86bacf4f56cbc6ace5b2df8138/pyscf/scf/dispersion.py#L145-L149
`pip install pyscf-dispersion` should work for pyscf that is installed by pip, however, I am using a fork and I build pyscf from source. Do I just put pyscf-dispersion as an extension? Do I need to build it?

I also found the description from https://github.com/pyscf/dispersion/blob/fe538a3af411140fcb84f25f901726215839b3ff/README.md
> If PyTorch is not used in your program, you can still install the multi-threading releases through pip install dftd3 dftd4
This looks like if PyTorch is not used, `pip install pyscf-dispersion` is not required and installing dftd3 is sufficient, but it turns out the actual code does not work like this.

**Comment by sunqm** [comment]
If you want to use keyword like `xc='wb97x-d3'`, pyscf-dispersion should be installed. Otherwise, you can use the simple-dftd3 package, and manually call the simple-dftd3 API to compute the dispersion contribution and add them to the SCF results.

The pyscf-dispersion module is an extension module. It works like other extensions, as described in the documentation 
https://pyscf.org/user/extensions.html#environment-variable-pyscf-ext-path

In short, you can clone the pyscf-dispersion repo and use cmake to compile the Fortran code. Then add the pyscf-dispersion directory of the environment variable  `PYSCF_EXT_PATH`

**Comment by YuzheZhang98** [comment]
@sunqm Is there any guidance to compile the dispersion extension? I tried building with setup.py but failed. Attached is the log file.
[compile.log](https://github.com/user-attachments/files/16860832/compile.log)

**Comment by tberkel** [comment]
We should remember to update the docs on DFT dispersion
https://pyscf.org/user/dft.html#dispersion-corrections and the examples
(`dft
/16-dft_d3.py`)

**Comment by verena-neufeld** [comment]
This might be marked as `feature request`?
